README.rst

```diff
@@ -95,6 +99,18 @@ To run javascript linter watcher:
 
 ``gulp lint:watch``
 
+To run javascript code style analysis:
+
+``gulp jscs``
```

Comment:
why have jscs separated from linting? fix can be a separate task, but jscs and jshint should be both part of linting, similar to what we have in djangoCMS